game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -121,14 +138,31 @@ class Cell(val id: Int, player: Player, var position: Vector2 = new Vector2(0f,
     targetVelocity - velocity
   }
 
+  def movement(deltaSeconds: Float): Vector2 = {
+    steering.truncate(maxSpeed) * deltaSeconds
```

Comment:
What does this line do? Is it to allow the `cell` to exceed the `maxSpeed` limit?